packages/communication-react/review/communication-react.api.md

```diff
@@ -123,22 +123,22 @@ export type AzureCommunicationCallAdapterArgs = {
     locator: CallAdapterLocator;
 };
 
-// @public
-export type AzureCommunicationChatAdapterArgs = {
+// @beta
+export type AzureCommunicationCallAndChatAdapterArgs = {
     endpoint: string;
     userId: CommunicationUserIdentifier;
     displayName: string;
     credential: CommunicationTokenCredential;
-    threadId: string;
+    callAndChatLocators: CallAndChatLocator | TeamsMeetingLinkLocator;
```

Comment:
very open to what this should be called (locator for the ~meeting~ CallAndChat composite

Reply:
does this locator also hold the threadId for the chat composite?

Reply:
Yeah, it's either an { ACS call locator + ACS chat thread ID }, or a teams meeting url

Reply:
Updated to `locator`